float64 brick_x
float64 brick_y
float64 brick_z

float64 brick_theta

float64 goal_x
float64 goal_y
float64 goal_z

float64 goal_theta
---
bool success
